doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -43,49 +43,110 @@ Increment the version number to the milestone (e.g. 5.10.1) in the following two
+- Under "Release title" use the same name as the tag such as ``v5.10.1``.
+- In the description, copy and paste the content from the release notes .md file created in the "Write Release Notes" steps above.
+- Click "Save draft" because we do not want to publish the release yet.
+
+At this point you can send around the draft release for any final feedback. Links to the guides for this release should be working now, since you build them above.
+
+Make corrections to the draft, if necessary. It will be out of sync with the .md file, but that's ok (`#7988 <https://github.com/IQSS/dataverse/issues/7988>`_ is tracking this).
+
+Run a Build to Create the War File
+----------------------------------
 
-The "tag version" and "title" should be the number of the milestone with a "v" in front (i.e. v5.10.1).
+ssh into the dataverse-internal server and undeploy the current war file.
 
-Copy in the content from the .md file created in the "Write Release Notes" steps above.
+Go to https://jenkins.dataverse.org/job/IQSS_Dataverse_Internal/ and make the following adjustments to the config:
+
+- Repository URL: ``https://github.com/IQSS/dataverse.git``
+- Branch Specifier (blank for 'any'): ``*/master``
+- Execute shell: Update version in filenames to ``dataverse-5.10.1.war`` (for example)
+
+Click "Save" then "Build Now".
+
+The build number will appear in ``/api/info/version`` (along with the commit mentioned above) from a running installation (e.g. ``{"version":"5.10.1","build":"907-b844672``).
+
+Build Installer (dvinstall.zip)
+-------------------------------
+
+ssh into the dataverse-internal server and do the following:
+
+- In a git checkout of the dataverse source switch to the master branch and pull the latest.
+- Copy the war file from the previous step to the ``target`` directory in the root of the repo (create it, if necessary).
+- ``cd scripts/installer``
+- ``make``
+
+A zip file called ``dvinstall.zip`` should be produced.
 
 Make Artifacts Available for Download
 -------------------------------------
 
 Upload the following artifacts to the draft release you created:
 
-- war file (``mvn package`` from Jenkins)
-- installer (``cd scripts/installer && make``)
-- other files as needed, such as updated Solr schema and config files
+- the war file (e.g. ``dataverse-5.10.1.war``, from above)
+- the installer (``dvinstall.zip``, from above)
+- other files as needed:
+
+  - updated Solr schema
+  - metadata block tsv files
+  - config files
 
 Publish the Release
 -------------------
 
 Click the "Publish release" button.
 
+Update Guides Link
+------------------
+
+"latest" at https://guides.dataverse.org/en/latest/ is a symlink to the directory with the latest release. That directory (e.g. ``5.10.1``) was put into place by the Jenkins "guides" job described above.
```

Comment:
Perhaps you could create a link to the specific section, instead of writing "described above"?